File: Demo.API/Demo.Api.Contracts/Models/Patient.cs
using System;

namespace Demo.Api.Contracts.Models
{
    public class Patient
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string SurName { get; set; }
        public DateTime DOB { get; set; }
        public string Gender { get; set; }
        public int CityId { get; set; }
    }
}

File: Angular App/src/app/models/patient.ts
export interface IPatient {
    Id: number;
    SurName: string;
    Name: string;
    DOB: Date;
    Gender: string;
    CityId: number;
}

export class Patient implements IPatient {
    constructor(public Id: number,
    public SurName: string,
    public Name: string,
    public DOB: Date,
    public Gender: string,
    public CityId: number){}
}
